components/src/core/components/Icon/Icon.vue

```diff
@@ -1,5 +1,6 @@
 <template>
-  <i :class="classes"></i>
+  <span v-if="ohrmIcon" :class="ohrmClasses">{{ name }}</span>
+  <i v-if="!ohrmIcon" :class="classes"></i>
```

Comment:
Suggestion, Since Ligatures support is with `<span>` tag. why not use it for both scenarios. 
`<span :class="classes">{{ this.ohrmIcon ? name:  ''  }}</span>`

then in computed we can
```
classes(): object {
  return {
    'oxd-icon': true,
    'oxd-material-icons': this.ohrmIcon
    [`bi-${this.name}`]: !this.ohrmIcon,
  };
}
```